automation/test.sh

```diff
@@ -40,6 +40,9 @@ if [[ $TARGET =~ windows.* ]]; then
 elif [[ $TARGET =~ cnao ]]; then
   export KUBEVIRT_WITH_CNAO=true
   export KUBEVIRT_PROVIDER=${TARGET/-cnao/}
+elif [[ $TARGET =~ sig-network ]]; then
```

Comment:
Why the `[]` are not included? Such a math may catch things lik `sig-networking`, no?

Reply:
Its the `TARGET`, which comes from the prow job env var, its not the `[sig-network]` label if that what you means,

Reply:
Ahh right, this comment refers to line 291 then.

Reply:
we never FOCUS `[X]`, but X and the test label either X or `[X]`
i don't think we should focus otherwise ?

Reply:
I do not understand you response. We have a specific label with square parenthesis and I gave an explicit example which may math as well (or not, you tell me).

Reply:
its a regex so it might catch sig-networking as well i believe,
but i meant that i don't see it as a problem, the labeling should be correct,
and we never used FOCUS that includes the `[]`

Reply:
Done (in a commit that i will rearrange in the end)